src/commands/user.command.ts

```diff
@@ -0,0 +1,66 @@
+import { Command } from "../deps.ts";
+import { setupLogger } from "../logger.ts";
+import { CmdGlobalOptions, OutputFormatType } from "./cmd-options.ts";
+import { CLICommand, loadConfig } from "../config/config.model.ts";
+import { verifyCliAvailability } from "../init.ts";
+import { MeshAdapterFactory } from "../mesh/mesh-adapter.factory.ts";
+import { MeshTableFactory } from "../presentation/mesh-table-factory.ts";
+import { TenantIamPresenterFactory } from "../presentation/tenant-iam-presenter-factory.ts";
+import { isatty } from "./tty.ts";
+import { MeshTenantFilterFactory } from "../mesh/filter/mesh-tenant-filter-factory.ts";
+
+export function registerUserCommand(program: Command) {
+  const userCmd = new Command()
+    .description(
+      `Enables you to list tenants based on a user. Run "${CLICommand} user -h" to see what is possible.`,
+    )
+    .action(() => {
+      userCmd.showHelp();
+    });
+  program.command("user", userCmd);
+
+  const listUserTenants = new Command()
+    // type must be added on every level that uses this type. Maybe bug in Cliffy?
+    .type("output", OutputFormatType)
+    .description(
+      "Returns a list of tenants with their name, id, tags and platform.",
+    )
+    .example(
+      "List all tenants who have the user john.doe@example.com assigned.",
+      `${CLICommand} user list principalName=john.doe@example.com`,
+    )
+    .action(listUserTenantAction);
+
+  userCmd
+    .command("list <filter:string>", listUserTenants);
```

Comment:
yeah now since I typed it a few times I have to admit I start to dislike the filter as well :D I mean it could make sense later if you want to filter for other properties but is totally annoying to type it every time.

Reply:
Should we then go with my proposal for now? We can always revisit the decision later.